src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h

```diff
@@ -162,8 +168,20 @@ namespace OpenMS
     OPENMS_DLLAPI void extractFirst(const std::vector<std::pair<double, double> >& peaks, std::vector<double>& mass);
     /// extract second from vector of pairs
     OPENMS_DLLAPI void extractSecond(const std::vector<std::pair<double, double> >& peaks, std::vector<double>& mass);
-    
+
+
+    /**
+      @brief Helper function for integrating a spectrum.
+    */
+    void integrateWindow_(const OpenSwath::SpectrumPtr& spectrum,
```

Comment:
missing `OPENMS_DLLAPI` prefix.
Same for OpenMS::DiaPrescore::score

Reply:
this  causes these build error https://cdash.openms.de/viewBuildError.php?buildid=148434

Reply:
Great thank you!